randovania/games/dread/gui/dialog/dread_connector_prompt_dialog.py

```diff
@@ -38,3 +40,13 @@ def text_value(self) -> str:
             return "localhost"
         else:
             return self.prompt_edit.text().strip()
+
+
+class CSConnectorPromptDialog(DreadConnectorPromptDialog):
+    def __init__(self, **kwargs: dict[str, Any]) -> None:
+        super().__init__(**kwargs)
+
+        self.ryujinx_radio.setText("Standard")
+        self.ryujinx_label.setText("Connects to a copy of Cave Story running on this computer.")
```

Comment:
Ah yes, running Freeware Cave Story in Ryujinx.

Reply:
it's reasonable code reuse!